src/VisualStudio/Core/Def/ColorSchemes/ColorSchemeApplier.cs

```diff
@@ -56,6 +61,11 @@ internal sealed partial class ColorSchemeApplier
         _settings = new ColorSchemeSettings(threadingContext, _serviceProvider, globalOptions);
         _colorSchemes = ColorSchemeSettings.GetColorSchemes();
         _classificationVerifier = new ClassificationVerifier(threadingContext, fontAndColorStorage, _colorSchemes);
+        _workQueue = new(
+            TimeSpan.FromMilliseconds(1000),
+            async (token) => await QueueColorSchemeUpdateAsync().ConfigureAwait(false),
```

Comment:
```suggestion
            async _ => await QueueColorSchemeUpdateAsync().ConfigureAwait(false),
```

Reply:
or, just replace this entirely with `QueueColorSchemeUpdateAsync` as the entire argument.